src/shared/watchdog.c

```diff
@@ -31,6 +31,7 @@
 #include "watchdog.h"
 
 static int watchdog_fd = -1;
+static char *watchdog_dev = NULL;
```

Comment:
I'd not abbreviate this either, just call it `watchdog_device`, so that the same name is used everywhere: externally, internally, in functions, in global variables, and so on.